ci/slow.go

```diff
@@ -0,0 +1,25 @@
+package ci
+
+import (
+	"os"
+	"strconv"
+	"testing"
+)
+
+// SkipSlow skips a slow test unless NOMAD_SLOW_TEST is set to a true value.
+func SkipSlow(t *testing.T, reason string) {
+	value := os.Getenv("NOMAD_SLOW_TEST")
+	run, err := strconv.ParseBool(value)
+	if !run || err != nil {
+		t.Skipf("Skipping slow test: %s", reason)
+	}
+}
+
+// Parallel runs t in parallel, unless CI is set to a true value.
```

Comment:
Yes! I made it! You have no idea how hard Chrome had to work to let me add a comment here. After minutes of loading and many seconds to pop up this textbox after I clicked the line, my typing is limited to about 0.5 character / s.

Anyway... mind adding a comment here about *why* we disable parallel for CI? I assume the answer is "because we manually parallelize tests across multiple CI jobs" but it would be nice to leave a comment for future generations who find their way here (and perhaps a code change or move has obliterated this PR from `git blame`)

Reply:
fixed

And thank you for your sacrifice!